coi/src/stats.rs

```diff
@@ -71,157 +71,161 @@ impl NegativeCoi {
 /// Computes the relevances of the positive cois.
 ///
 /// The relevance of each coi is computed from its view count and view time relative to the
-/// other cois. It's an unnormalized score from the interval `[0, ∞)`.
+/// other cois and ranges in the interval `[0., 2.]`.
 pub fn compute_coi_relevances<'a>(
     cois: impl IntoIterator<IntoIter = impl Clone + Iterator<Item = &'a PositiveCoi>>,
     horizon: Duration,
     time: DateTime<Utc>,
 ) -> Vec<f32> {
     let cois = cois.into_iter();
-    #[allow(clippy::cast_precision_loss)] // small values
-    let view_counts =
-        cois.clone().map(|coi| coi.stats.view_count).sum::<usize>() as f32 + f32::EPSILON;
+
+    let view_counts = cois.clone().map(|coi| coi.stats.view_count).sum::<usize>();
+    #[allow(clippy::cast_precision_loss)] // acceptable for large values
```

Comment:
maybe my comment on the lint was to brief. what i was trying to say it not that it in general acceptable to loose precision on the cast, but that in this case we don't care much if there are precision losses for high values.

if we do care we should handle that case, but until now we didn't.

Reply:
True, if the same coi is used for a long time the value gets less precise, we just don't care because this precision loss doesn't affect the algorithm in a relevant way . 

maybe it's best to not have an comment here :thinking: